openhtf/util/validators.py

```diff
@@ -75,9 +76,30 @@ def create_validator(name, *args, **kwargs):
 
 _identity = lambda x: x
 
+
+class ValidatorBase(object):
```

Comment:
Added some documentation clarifying that validators do not need to be classes, but if being implemented as such, there are benefits to inheriting them from the base classes.